yaml/csr_template.yaml

```diff
@@ -36,7 +37,7 @@
 #    - MSB_FIELD_NAME:
 #      - description: >
 #          BRIEF_DESCRIPTION
-#      - type: TYPE (WPRI/WLRL/WARL)
+#      - type: TYPE (WPRI/WLRL/WARL/R)
```

Comment:
I think you are using mixed terminologies here. We should probably change to standard CSR terminology like WR/RO/W0C/W1C etc. And for WR field, we should specify the allowed values, rather than WARL which doesn't tell too much what's allowed. The allowed values could be done in a later PR.

Reply:
marking resolved for now